Algorithm.CSharp/MeanVarianceOptimizationAlgorithm.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections.Generic;
+using QuantConnect.Algorithm.Framework;
+using QuantConnect.Algorithm.Framework.Alphas;
+using QuantConnect.Algorithm.Framework.Execution;
+using QuantConnect.Algorithm.Framework.Portfolio;
+using QuantConnect.Algorithm.Framework.Risk;
+using QuantConnect.Algorithm.Framework.Selection;
+using QuantConnect.Orders;
+using QuantConnect.Interfaces;
+using System.Linq;
+using QuantConnect.Data.UniverseSelection;
+using QuantConnect.Algorithm.Framework.Portfolio.Optimization;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    public class MeanVarianceOptimizationAlgorithm : QCAlgorithmFramework, IRegressionAlgorithmDefinition
+    {
+        IEnumerable<Symbol> _symbols = (new string[] { "AIG", "BAC", "IBM", "SPY" }).Select(s => QuantConnect.Symbol.Create(s, SecurityType.Equity, Market.USA));
+
+        /// <summary>
+        /// Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.
+        /// </summary>
+        public override void Initialize()
+        {
+            // Set requested data resolution
+            UniverseSettings.Resolution = Resolution.Minute;
+            int lookback = 1;
+
+            SetStartDate(2013, 10, 07);  //Set Start Date
+            SetEndDate(2013, 10, 11);    //Set End Date
+            SetCash(100000);             //Set Strategy Cash
+
+            // Find more symbols here: http://quantconnect.com/data
+            // Forex, CFD, Equities Resolutions: Tick, Second, Minute, Hour, Daily.
+            // Futures Resolution: Tick, Second, Minute
+            // Options Resolution: Minute Only.
+            
+            // set algorithm framework models
+            SetUniverseSelection(new CoarseFundamentalUniverseSelectionModel(CoarseSelector));
+            SetAlpha(new HistoricalReturnsAlphaModel(lookback, Resolution.Daily));
+            SetPortfolioConstruction(new MeanVarianceOptimizationPortfolioConstructionModel(new MeanVariancePortfolio()));
```

Comment:
In the python version, `MeanVarianceOptimizationPortfolioConstructionModel`'s first parameter is optional. We think that the default value should be this (`MeanVariancePortfolio`) optimization method.

Also the field variable `_symbols` should have the **private** access modifier.